apps/designer/app/designer/features/sidebar-left/panels/components/image.tsx

```diff
@@ -18,18 +26,17 @@ const useImageWithFallback = ({ path }: { path: string }) => {
   return src;
 };
 
-export const AssetManagerImage = ({
-  path,
-  alt,
-  status,
-}: {
-  path: string;
-  alt?: string;
-  status?: Asset["status"];
-}) => {
+export const AssetManagerImage = ({ path, alt, status, name, id }: Asset) => {
+  const submit = useSubmit();
   const isUploading = status === "uploading";
   const src = useImageWithFallback({ path });
+  const [isToolTipOpen, setTolltipOpen] = useState(false);
   const [progressBarPercentage, setProgressBarPercentage] = useState(0);
+  const tooltipRef = useRef(null);
+  useClickAway(tooltipRef, () => {
+    setTolltipOpen(false);
+  });
+  useKey("Escape", () => setTolltipOpen(false));
```

Comment:
I wonder if we should use consistently useHotkeys for this

Reply:
definitly, updated